src/Bicep.LangServer/Handlers/BicepDidChangeWatchedFilesHandler.cs

```diff
@@ -55,17 +55,12 @@ protected override DidChangeWatchedFilesRegistrationOptions CreateRegistrationOp
                     new FileSystemWatcher()
                     {
                         Kind = WatchKind.Create | WatchKind.Change | WatchKind.Delete,
-                        GlobPattern = new("**/")
+                        GlobPattern = new("**")
                     },
                     new FileSystemWatcher()
                     {
                         Kind = WatchKind.Create | WatchKind.Change | WatchKind.Delete,
-                        GlobPattern = new("**/*.bicep")
-                    },
-                    new FileSystemWatcher()
-                    {
-                        Kind = WatchKind.Create | WatchKind.Change | WatchKind.Delete,
-                        GlobPattern = new("**/*.{json,jsonc,arm}")
+                        GlobPattern = new("**/*")
```

Comment:
I may not understand the nuance here, but is it possible that both of these watchers are doing the same thing?